js/index.js

```diff
@@ -1,7 +1,50 @@
 // Iteration 1: Names and Input
-
+let firstHacker = "Carmen";
+console.log("The driver's name is", firstHacker);
+let secondHacker = "Andrea";
+console.log("The navigator's name is", secondHacker);
```

Comment:
Para imprimir una variable en la misma cadena, pueden usar:
* concatenación => `console.log("The driver's name is" + firstHacker);`
* interpolación de cadena => `console.log(`The driver's name is ${firstHacker}`);`
Usar interpolación siempre que se puede ;)